catalog/core/catalog-core-commands/pom.xml

```diff
@@ -129,10 +129,12 @@
                             ddf.catalog.data.impl,
                             ddf.catalog.operation.impl,
                             ddf.catalog.util.impl,
-                            ddf.catalog.filter.impl
+                            ddf.catalog.filter.impl,
+                            ddf.catalog.source.impl
                         </Private-Package>
                         <Export-Package />
                         <Import-Package>
```

Comment:
Why in the world is this getting pulled in and needs to be excluded? Doesn't the dependency have a `<scope>test</scope>`?